samples/extensions/vertex_dynamic_state/vertex_dynamic_state.h

```diff
@@ -0,0 +1,101 @@
+/* Copyright (c) 2022, Mobica Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "api_vulkan_sample.h"
+
+class vertex_dynamic_state : public ApiVulkanSample
```

Comment:
I think, the naming conventions here would make this class be named `VertexDynamicState`, instead of `vertex_dynamic_state`.